release.nix

```diff
@@ -40,18 +40,27 @@
 
 with (import pkgs.iohkNix.release-lib) {
   inherit pkgs;
-  inherit supportedSystems supportedCrossSystems scrubJobs projectArgs;
+  supportedSystems = [ "x86_64-linux" ];
```

Comment:
For clarity, could this release-lib call be put alongside the macos one?
So have two variables: `linuxRelease` and `macosRelease`.

Reply:
Funnily enough I had both `linuxRelease` and `darwinRelease` at first to make sure I found all the places things needed to be changed to `darwinRelease`.  Then I stripped out `linuxRelase` to make the PR smaller.